doc-surrealdb_versioned_docs/version-2.x/surrealql/statements/define/function.mdx

```diff
@@ -75,6 +80,32 @@ RETURN fn::last_option(1);
 };
 ```
 
+## Permissions
+
+You can set the permissions for a custom function using the `PERMISSIONS` clause. The `PERMISSIONS` clause is mostly used to restrict who can access a function and what data they can access. It can be set to `NONE`, `FULL`, or `WHERE @condition`.
+
+- `FULL`: When Full permissions are granted [record](/docs/surrealdb/2.x/security/authentication#record-users) users have access to the function. This is the default permission when not specified.
+- `NONE`: When this permission is granted, [record](/docs/surrealdb/2.x/security/authentication#record-users) users  have no access to the defined function.
+- `WHERE @condition`: Permissions are granted to the function based on the specified condition.
+
+### Using the `FULL` permission
+
+The `FULL` permission grants all users access to the function. The following example defines a function that fetches all products from the `product` table and grants the function full permissions to access the data to all users.
```

Comment:
Recommend linking to the blog post so curious users can see where all the neat data is coming from:

from the `product` table -> From the `product` table of the [Surreal Deal Store dataset](https://surrealdb.com/blog/our-new-demo-dataset-has-a-lot-in-store-for-you)

Reply:
Added a note